user_interface_components/dialog_wrapper.md

```diff
@@ -70,3 +70,50 @@ Otherwise, you need to return a
 class which encapsulates an error message and an optional component associated with the invalid data.
 If you specify a component, an error icon will be displayed next to it, and it will be focused when the user tries to invoke the `OK` action.
 
+## Example
+
+Simple definition of a
+[DialogWrapper](upsource:///platform/platform-api/src/com/intellij/openapi/ui/DialogWrapper.java):
+
+```java
+package com.example.plugin.component;
+
+import com.intellij.openapi.ui.DialogWrapper;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class SampleDialogWrapper extends DialogWrapper {
+
+    public SampleDialogWrapper() {
+        super(true); // use current window as parent
+        super.init();
```

Comment:
No need to use an explicit "super" qualifier.

Reply:
you're right thx